Add backend error linking to replays (#8389)
getsentry/sentry-docs: src/docs/product/session-replay/index.mdx

Session Replay shows you a video-like reproduction of your user sessions so you can see what happened before, during, and after an error or performance issue occurred. Gain deeper debugging context into issues so that you can reproduce and resolve problems faster without the guesswork. As you play back each session, you can see every user interaction in relation to network requests, DOM events, and console messages. It’s effectively like having [DevTools](https://developer.chrome.com/docs/devtools/overview/) active in your production user sessions.

> Replays are integrated with other areas of the Sentry product so you can see how the user experience is impacted by errors and slow transactions. Replays associated with error events will appear on the [Issue Details](/product/issues/issue-details/) page, while those showing slow transactions will appear on the [Transaction Summary](/product/performance/transaction-summary/) page. If you have [backend error linking](/product/session-replay/getting-started#backend-error-linking) set up, any contributing backend errors are included in the replay's timeline, breadcrumbs, and errors.

Reviewer: ```suggestion
Replays are integrated with other parts of the Sentry product so you can see how the user experience is impacted by errors and slow transactions. You'll see replays associated with error events on the [Issue Details](/product/issues/issue-details/) page, and those associated with slow transactions on the [Transaction Summary](/product/performance/transaction-summary/) page. If you have [backend error linking](/product/session-replay/getting-started#backend-error-linking) set up, any contributing backend errors will be included in the replay's timeline, breadcrumbs, and errors.
```
